docs/zh-tw/components/helpers.md

```
@@ -1,41 +1,99 @@
+## 設定與自動載入
+
+此套件透過 Composer 自動載入函式檔案。其設定提供器不會註冊或發布任何設定，因此無需額外設定。
+
+除 `call` 定義在 `FriendsOfHyperf\Helpers\Command` 命名空間外，其他函式均定義在
+`FriendsOfHyperf\Helpers` 命名空間。
+
+## 函式參考
+
+| 函式 | 簽名與行為 |
+| --- | --- |
+| `app` | `app(null|string|callable $abstract = null, array $parameters = [])`：從容器解析服務；將可呼叫值轉換為 `Closure`。 |
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)